presto-main/src/main/java/com/facebook/presto/sql/planner/plan/GroupIdNode.java

```diff
@@ -84,6 +90,12 @@ public PlanNode getSource()
         return groupingSets;
     }
 
+    @JsonProperty
+    public Map<Symbol, Symbol> getPassthroughSymbolMap()
```

Comment:
rename to `mappings` or `inputToOutputMappings`/`outputToInputMappings` (depending on what this represents)